<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowTotalRevenues</class>
 <widget class="QDialog" name="ShowTotalRevenues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listar Faturamento | Sistema de Controle de Comandas</string>
  </property>
  <widget class="QLabel" name="label_showTotalRevenuesTitle">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Listar Faturmento</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_showTotalRevenues">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Faturamento</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_searchPeriodTotalRevenues">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>351</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_showTotalRevenuesPeriod">
    <item>
     <widget class="QLabel" name="label_showTotalRevenuesPeriodTitle">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Selecionar período</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_showTotalRevenuesPeriod">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_showTotalRevenuesPeriodFromDate">
        <item>
         <widget class="QLabel" name="label_fromDate">
          <property name="text">
           <string>De: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_fromDate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_showTotalRevenuesPeriodToDate">
        <item>
         <widget class="QLabel" name="label_toDate">
          <property name="text">
           <string>Até: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_toDate"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
